src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -331,12 +336,16 @@ private class AppTaskLauncherActor(
   }
 
   private[this] def replyWithQueuedTaskCount(): Unit = {
+    val taskLaunchesInFlight =
+      inFlightTaskOperations
+        .keys
+        .count(taskId => tasksMap.get(taskId).exists(_.launched.isDefined))
     sender() ! QueuedTaskInfo(
       app,
       tasksLeftToLaunch = tasksToLaunch,
-      taskLaunchesInFlight = inFlightTaskOperations.size,
+      taskLaunchesInFlight = taskLaunchesInFlight,
```

Comment:
We need to add taskReservationsInFlight.
QueuedTaskInfo.inProgress should be adapted accordingly.